### DISCUSSION TITLE
Mini Cheetah跑速突破

### DISCUSSION THREAD

**Discussion by 11timyyy** [1101-GS4719-C/Python-Community-GS4719-C#502] [General]
以前所熟知的AI機器人大多為站姿，並利用雙腳走路。但是這篇文章卻在介紹有四隻腳的機器人。
近日麻省理工學院改善豹型機器人 Mini Cheetah 時，利用 AI 驅動的模擬系統調整工作，訓練機器人在不同狀況下調整不同步姿。也因此
Mini Cheetah生成新的跑姿，且達到 14.04 公里跑速新紀錄。那實際能幹嘛呢? 文章中提到可以運用在無人機飛行學習上，透過AI驅動，快速供給物資抑或是在戰爭上迅速觀察敵人的動向等等。

文章來源:https://technews.tw/2022/03/21/how-the-mit-mini-cheetah-learns-to-run/

**Comment by liangyinglly** [comment]
這隻mini cheetah說不定未來能發展具有導盲犬功能的能力或拿來訓練田徑運動員的工具

**Comment by sean1325418** [comment]
或許這技術可以給運動員研究怎樣的姿勢可以跑出更快的速度

**Comment by ghost** [comment]
畢竟四輪對著地性與平衡要求相較於二輪是有它的優勢在

**Comment by ThomasMeng0515** [comment]
有一家公司叫做波士頓動力(Boston Dynamics)之前也有做過四隻腳走路的機器人，甚至可以做前翻、後翻的動作。現在的機器人功能真的日新月異，進步很快

**Comment by hapiwangy** [comment]
四足機器人比起兩足的機器人在平衡的方面更為安全，但是在移動的時候比起雙足的機器人可能就需要有更多的協調性。期待未來能了解到更多的相關技術。

**Comment by cordydeng** [comment]
或許未來能夠在應用在維護社會治安方面 , 像是抓壞人之類的

**Comment by PoooYen** [comment]
或許未來有機會變成另類的行動監視器

**Comment by sys511613** [comment]
把AI結合上去真的是一大進步
但怎麼感覺他跑起來的樣子怪可怕的

**Comment by jcpan821** [comment]
以後打仗會不會變機器人大戰

**Comment by stan890425** [comment]
機器人功能越來越完善了

**Comment by alexthebigpp** [comment]
感覺可以讓大家拿去改裝，之後拿來賭博看誰跑比較快，像機械賽馬

**Reply by jason79461385** [reply]
機械賽馬感覺很酷 又是一個可以炒作的東西呢
